crt_portal/cts_forms/tests.py

```diff
@@ -442,14 +487,14 @@ def test_required_user_logging(self):
 
             self.assertEqual(
                 cm.check_present(
-                    ('cts_forms.signals', 'INFO', 'ADMIN ACTION by: CLI CLI @ CLI User saved: 5 permissions: <QuerySet []> staff: False superuser: False active: True')
+                    ('cts_forms.signals', 'INFO', 'ADMIN ACTION by: CLI CLI @ CLI User saved: 7 permissions: <QuerySet []> staff: False superuser: False active: True')
```

Comment:
This is the line I was talking about. I think we should have some mechanism to not have to manually update these permission checks as we add new tests.

Reply:
Yea, good idea- I added some formatting to get the user id dynamically.